src/Models/Entities/User.cs

```diff
@@ -9,5 +9,6 @@ namespace Models.Entities
 {
     public class User : IdentityUser
     {
+        public List<Favourites> Favorites { get; set; }
```

Comment:
same here

Reply:
that is the question

Reply:
Boris Johnson > Sleepy Joe
Boris Johnson approves u in Favourite

Reply:
Done